localisation/touhou_decisions_l_english.yml

```
@@ -633,5 +633,8 @@
  jnk_disable_manage_grudge_tt: "\"§YSupervise the Battle Personally§!\"'s effect will be disabled and battle result will no longer affect Junko's grudge." 
  jnk_grudge_mechanic_information_title: "Junko's Emotional Management"
  jnk_grudge_mechanic_information_desc: "Our [Root.Monarch.GetTitle] has unmeasurable grudge on Lunarian's deity Chang'e, and this grudge is still growing constantly. Although her purified grudge could strengthen our army, too high grudge will be dangerous. Therefore, it is our duty to manage our [Root.Monarch.GetTitle]'s emotion well." 
+ jnk_releases_grudge_title: "Release Grudge"
+ jnk_releases_grudge_desc: "We will induce Junko release her unbridled grudge towards the enemies of our realm."
```

This might flow a bit better? Also "unbridled grudge" feels weird, maybe could be "unbridled furies" or "unbridled rage", not sure. Doesn't have to change though.
```suggestion
 jnk_releases_grudge_desc: "We will have Junko release her unbridled grudge towards the enemies of our realm."
```